src/modding/ModManager.cs

```diff
@@ -857,6 +1308,251 @@ private void OpenInfoUrlPressed()
+        {
+            return;
+        }
+
+        otherModInfoDialog.WindowTitle = TranslationServer.Translate("MOD_REQUIRED_MODS");
+        var infoText = string.Empty;
+
+        var currentModRequiredMods = selectedMod.Info.RequiredMods;
+        if (currentModRequiredMods != null)
+        {
+            foreach (string currentRequiredMod in currentModRequiredMods)
+            {
+                if (!string.IsNullOrWhiteSpace(currentRequiredMod))
+                {
+                    infoText += "* " + currentRequiredMod + "\n";
+                }
+            }
+        }
+        else
+        {
+            infoText += TranslationServer.Translate("NO_REQUIRED_MODS");
+        }
+
+        otherModInfoDialog.DialogText = infoText;
+        otherModInfoDialog.PopupCenteredShrink();
+    }
+
+    private void OnIncompatiblePressed()
+    {
+        GUICommon.Instance.PlayButtonPressSound();
+        if (selectedMod is null)
+        {
+            return;
+        }
+
+        otherModInfoDialog.WindowTitle = TranslationServer.Translate("INCOMPATIBLE_WITH");
+        var infoText = string.Empty;
+
+        var currentModIncompatibleMods = selectedMod.Info.IncompatibleMods;
+        if (currentModIncompatibleMods != null)
+        {
+            foreach (string currentIncompatibleMod in currentModIncompatibleMods)
+            {
+                if (!string.IsNullOrWhiteSpace(currentIncompatibleMod))
+                {
+                    infoText += "* " + currentIncompatibleMod + "\n";
+                }
+            }
+        }
+        else
+        {
+            infoText += TranslationServer.Translate("NO_MOD_INCOMPATIBLE");
+        }
+
+        otherModInfoDialog.DialogText = infoText;
+        otherModInfoDialog.PopupCenteredShrink();
+    }
+
+    private void OnLoadOrderPressed()
+    {
+        GUICommon.Instance.PlayButtonPressSound();
+        if (selectedMod is null)
+        {
+            return;
+        }
+
+        otherModInfoDialog.WindowTitle = TranslationServer.Translate("LOAD_ORDER");
+        var infoText = string.Empty;
+
+        var currentModLoadBefore = selectedMod.Info.LoadBefore;
+        var currentModLoadAfter = selectedMod.Info.LoadAfter;
+        if (currentModLoadBefore != null || currentModLoadAfter != null)
+        {
+            if (currentModLoadAfter != null)
+            {
+                infoText += TranslationServer.Translate("MOD_LOAD_AFTER") + "\n";
+                foreach (string currentLoadAfterMod in currentModLoadAfter)
+                {
+                    if (!string.IsNullOrWhiteSpace(currentLoadAfterMod))
+                    {
+                        infoText += "* " + currentLoadAfterMod + "\n";
+                    }
+                }
+            }
+
+            // If there both 'load before' and 'load after' is going to display add a empty line between them
+            if (currentModLoadBefore != null && currentModLoadAfter != null)
```

Comment:
Elsewhere I've used code where we just check if the string we are adding to is null.
Btw now I just realized that you do string *building* with normal strings. That's bad for performance, please switch to using StringBuilder in these situations.